charts/draft-store-service/values.preview.template.yaml

```diff
@@ -0,0 +1,8 @@
+java:
+  environment:
+    DRAFT_STORE_DB_HOST: ${SERVICE_NAME}-postgresql  # do not modify
+    IDAM_URL: https://idam-api.aat.platform.hmcts.net
```

Comment:
may as well put idam url and s2s url in the defaults? and not duplicate across the preview and aat files?